src/Compilers/CSharp/Portable/Binder/Semantics/OverloadResolution/OverloadResolution.cs

```diff
@@ -1252,9 +1252,12 @@ private static TypeSymbol GetParameterType(int argIndex, MemberAnalysisResult re
                                                        out okToDowngradeToNeither);
                 }
 
+                var type1Normalized = type1.NormalizeTaskTypes(Compilation);
+                var type2Normalized = type2.NormalizeTaskTypes(Compilation);
```

Comment:
What happens in the case where `Task` and `Task<T>` are not available?  Does it break our ability to do overload resolution on other task-like types?